test/ui-automation/test/specs/credential-interaction-flow.js

```diff
@@ -468,21 +468,21 @@ describe("TrustBloc Wallet - Store/Share credential flow (CHAPI)", () => {
 
     await browser.navigateTo(`${browser.config.walletURL}/credentials`);
 
-    for (const [key, value] of credential.entries()) {
-      console.log("validate vc in wallet : start ", key);
+    // for (const [key, value] of credential.entries()) {
```

Comment:
instead of commenting these lines, we can add `it.skip` to line 466.

Reply:
We actually need line 469 to execute to redirect to the credentials screen in wallet